controllers/storagecluster/storageclasses.go

```diff
@@ -227,7 +228,7 @@ func newCephFilesystemStorageClassConfiguration(initData *ocsv1.StorageCluster)
 					"description": "Provides RWO and RWX Filesystem volumes",
 				},
 			},
-			Provisioner:   fmt.Sprintf("%s.cephfs.csi.ceph.com", initData.Namespace),
+			Provisioner:   fmt.Sprintf("%s.cephfs.csi.ceph.com", os.Getenv(util.OperatorNamespaceEnvVar)),
```

Comment:
instead of calling os.GetEnv, write a helper function in util which returns the namespace and use it?

Reply:
We already have a func in util. But that also returns an error which I don't want to check here. We are already checking the error in main.
https://github.com/red-hat-storage/ocs-operator/blob/main/main.go#L123

Reply:
if you really want you can write a wrapper in util that skips the return error and even caches the operator namespace and return it or you can store it in the reconcile struct as well. calling os.Getenv from the main controller multiple times doesn't look good as this value doesn't get changed anytime during runtime.

Reply:
Yes I am planning to cache it in further changes. I am making small small changes which makes it easier to test the things.